docs: remove link to QueryTableResult (#218)
influxdata/influxdb-client-go: README.md

```

### Queries

> Query client offers two ways of retrieving query results, parsed representation in QueryTableResult and a raw result string. 

Reviewer: ```suggestion
Query client offers retrieving of query results to a parsed representation in a `QueryTableResult` or to a raw result string. 
```
